media-gfx/dmtx-utils/dmtx_utils-0.7.4.recipe

```diff
@@ -0,0 +1,62 @@
+SUMMARY="A software for reading and writing Data Matrix barcodes"
+DESCRIPTION="This is open source software for reading and writing Data Matrix \
+barcodes on Linux, Unix, OS X, Windows, and mobile devices. At its core \
+libdmtx is a native shared library, allowing C/C++ programs to use \
+its capabilities without extra restrictions or overhead."
+HOMEPAGE="http://libdmtx.sourceforge.net/"
+COPYRIGHT="2009-2017 Mike Laughton"
+LICENSE="GNU LGPL v2"
+REVISION="1"
+SOURCE_URI="https://downloads.sourceforge.net/libdmtx/dmtx-utils-$portVersion.tar.gz"
+CHECKSUM_SHA256="27104dba2a53979c30a0788f4c9ccbf5e30fbe382b9bb6942928f9f869f528b6"
+SOURCE_DIR="dmtx-utils-$portVersion"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+if [ "$targetArchitecture" != x86_gcc2  ]; then
+	commandBinDir=$binDir
+else
+	commandBinDir=$prefix/bin
+fi
+
+PROVIDES="
+	dmtx_utils$secondaryArchSuffix = $portVersion
+	cmd:dmtxquery = $portVersion
+	cmd:dmtxread = $portVersion
+	cmd:dmtxwrite = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libdmtx$secondaryArchSuffix
+	lib:libgomp$secondaryArchSuffix
+	lib:libMagickWand_6.Q16$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	imagemagick${secondaryArchSuffix}_devel
```

Comment:
Please change BUILD_REQUIRES to
```
BUILD_REQUIRES="
	haiku${secondaryArchSuffix}_devel
	devel:libdmtx$secondaryArchSuffix
	devel:libMagickCore_6.Q16$secondaryArchSuffix
	devel:libMagickWand_6.Q16$secondaryArchSuffix
	"
```
and insert
```
	lib:libMagickCore_6.Q16$secondaryArchSuffix
```
in REQUIRES just above
```
	lib:libMagickWand_6.Q16$secondaryArchSuffix
```

How did I find that we also need libMagickCore ?
Easy: I commented out all the lib:libMagick\* in REQUIRES and then called haikuporter in _strict_ mode, that is, with the "-S" option.
You should try it to see how it behaves when you have missing dependencies.